pyblish_standalone/__main__.py

```diff
@@ -22,6 +22,11 @@ def cli():
                               "can be called multiple times"))
     parser.add_argument("-rh", "--register-host", action="append",
                         help=("Append hosts to register."))
+    parser.add_argument("-rg", "--register-gui", action="append",
+                        help=("Append guis to register."))
+    parser.add_argument("--debug", action="store_true")
+    parser.add_argument("--port", nargs='?',
```

Comment:
You should ignore this PR atm, I've been doing experiments with AE and uploaded to my repo without knowing there was an ongoing PR.

The ```port``` is being used in pyblish-aftereffects [here](https://github.com/tokejepsen/pyblish-aftereffects/blob/03e78fa092e939b4feaf981ec9d52c080ae7f766/pyblish_aftereffects/util.py#L9), but I'm very certain I'll be removing it again and use environment variables instead.